src/panels/lovelace/common/is-object.js

```diff
@@ -0,0 +1,4 @@
+// Check if given obj is a JS object and optionally contains all required keys
+export default function isObject(obj, requiredKeys = []) {
```

Comment:
Let's not call this `isObject` because it does more

Reply:
`isValidObject`?  :man_shrugging: